Gemfile

```diff
@@ -151,7 +151,7 @@ group :development do
   # listen higher than 3.1.1 require Ruby version >= 2.2.3 (we're currently on
   # 2.2.2). Restrict the version of `listen` to prevent `guard-rspec`
   # introducing an incompatible dependency:
-  gem 'listen', '~> 3.0.5', '<= 3.1.1'
+  gem 'listen', '>= 3.0.5', '< 3.2'
```

Comment:
Once in ruby `2.4.1`, I'm see this warning when booting rails:
```
[...]/.rvm/gems/ruby-2.4.1/gems/sinatra-2.0.0.beta2/lib/sinatra/base.rb:1256: warning: constant ::Fixnum is deprecated
```

Do you see it?